### PR TITLE
Feat/socketRefactor

### PR DESCRIPTION
**Pull request by carlospatinos**
Can I merge this ? Basic refactoring to more meaningful names and small improvements to socket logic

### CODE REVIEW

File: packages/scrum-client/src/components/TeamList/TeamList.js [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-  // Object.keys(storyVotes).length === 0
-  // JSON.stringify(obj) === '{}';
-  // eslint-disable-next-line
-  console.log('Object.keys(storyVotes).length', Object.keys(storyVotes).length);
-  // eslint-disable-next-line
-  const storyVote = JSON.stringify(storyVotes) === '{}' ? undefined : storyVotes.find(([id]) => {
-          // eslint-disable-next-line
-    console.log('id', id);
+  const storyVote =
+    JSON.stringify(storyVotes) === '{}'
```
Comment (doncesarts) [inline packages/scrum-client/src/components/TeamList/TeamList.js]: JSON.stringify does convert to a string , not and object .. right ?
Is that comparison correct ?

File: packages/scrum-client/src/components/TeamList/TeamList.js [hunk 2]
```diff
@@ -24,6 +17,46 @@ const getUserVote = (storyVotes, user) => {
 
 const TeamList = props => {
   const { title, subtitle, users, storyVotes, admin, summaryVotes } = props;
+  const data = [
```
Comment (doncesarts) [inline packages/scrum-client/src/components/TeamList/TeamList.js]: is this data used?

File: packages/scrum-client/src/components/TeamList/TeamList.js [hunk 3]
```diff
@@ -41,11 +74,32 @@ const TeamList = props => {
...
+            <YAxis label={{ value: '# of votes', angle: -90, position: 'insideLeft' }} />
+            <Bar dataKey="numberOfVotes" fill="#8884d8" background={{ fill: '#eee' }}>
+              {' '}
+              <LabelList dataKey="voters" position="top" />
+            </Bar>
+          </BarChart>
+        </Col>
+        <Col sm={4}>
+          <Row>
+            <Col>Fastest responder: Luis</Col>
```
Comment (doncesarts) [inline packages/scrum-client/src/components/TeamList/TeamList.js]: pocho?

File: packages/scrum-server/bin/www [hunk 4]
```diff
@@ -37,4 +36,4 @@ function normalizePort(val) {
 var port = normalizePort(process.env.PORT || '3000');
 var server = HttpServer({ app, port, pjson });
 app.set('port', port);
-app.set('socketService', new SocketService(server));
+app.set('socketService', new ServerSingletonSocketService(server));
```
Comment (doncesarts) [inline packages/scrum-server/bin/www]: No soy muy fan de "ServerSingletonSocketService" especificar q es un singleton en el nombre ... 
"ServerSocketService" ? en caso de cambie en un furutro  ?

File: packages/scrum-client/src/components/TeamList/TeamList.js (hunk 3 above)
Comment (carlospatinos) [reply packages/scrum-client/src/components/TeamList/TeamList.js]: 😞

**Pull request merged**